Test.C

```diff
@@ -1,53 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 /*
 
-Hello
+defining number literals
 
 */
 
-int main(){ 
-
-    //initialize game 
+#define NAME_LENGTH 20
+#define DESCRIPTION_LENGTH 200
+#define NUM_EQUIPMENT 10
+#define NUM_ENCRYPTION 10
+#define NUM_ATTACK 5
+
+int main(){
+
+    //initialize game
+
+    struct Equipment {
+        int id;
+        char name[NAME_LENGTH]; // name of the piece of Equipment
+        char description[DESCRIPTION_LENGTH]; // short description of Equipment
+        int credits; // cost of the Equipment
+        int effectiveness; // number of messages able to be sent
```

Comment:
I like the use of similar vocab, but I do not think effectiveness really documents what this variable is. This is the quantity part of our quality vs quantity decision, so I think something like `production`, `capability`, or `messageAmount` would be more appropriate.
```suggestion
        int production; // number of messages able to be sent
```

Reply:
I think `messageAmount` might be more clear here